refactor: apply feedback (#176)
Innoactive/Creator: .Documentation/articles/getting-started/designer.md

#### Training Course Creation Wizard

To create a training course, in the top menu bar, select the `Innoactive > Create New Course...` menu option. This will open the Training Creation Wizard. Type the name of your training and press the `Create` button.

The created training course will be set as the selected one for the current scene.

> Learn more about the [Training Course Creation Wizard](../innoactive-creator/course-creation-wizard.md).

#### Workflow Window

> Now the [Workflow Window](../innoactive-creator/workflow-editor.md) is open and you can view and modify your training course. On the left, the list of chapters is displayed. You can use as many as you want, but let us stick to a single one in this tutorial. You can see the chapter workflow on the right. An empty chapter has only a starting point.

Reviewer: ```suggestion
Now [the Workflow window](../innoactive-creator/workflow-editor.md) is open and you can view and modify your training course. On the left, the list of chapters is displayed. You can use as many as you want, but let us stick to a single one in this tutorial. You can see the chapter workflow on the right. An empty chapter has only a starting point.
```

#### Training Course Creation Wizard

> To create a training course, in the top menu bar, select the `Innoactive > Creator > Create New Course...` menu option. This will open the Training Creation Wizard. Type the name of your training and press the `Create` button.

Reviewer 2: ```suggestion
To create a training course, in the top menu bar, select the `Innoactive > Create New Course...` menu option. This will open the Training Creation Wizard. Type the name of your training and press the `Create` button.
```

Reviewer 2: Can you check in the whole file where the path to the menu is still wrong? Thank you

Author: good catch! will do 👍

Author: I have checked this for every menu option, I am think I found them all

#### Step Inspector

> The [Step Inspector](../innoactive-creator/step-inspector.md) allows you to set a step name and description, as well as to create and edit behaviors, transitions, and conditions. The name and description of the step have no effect on the training course itself. Use them to keep notes for yourself and other trainers. To add a behavior, click on an `Add Behavior` button in the step's view. To delete a behavior, click on it's button with the trash icon. The same applies to transitions and conditions.

Reviewer 2: Liked the phrasing with "click on the trash button next to it" better, but doesn't matter too much

Author: I changed it because I did not like the next to it. I see the trash icon as part of it instead of it being next to it 🤔  but I can also leave it with the cenxt to it. What do you think?

Author: I found another description in the Step Inspector md, and I copied that one for consistency :)

> 1. Now, rename the first step of the training course to `Grab sphere`. In the `Transitions` tab, add a single `Grab Object` condition to its transition to the second step. Drag the previously created `Sphere` into the object field named `Grabbable object`. Add a description:

Reviewer: ```suggestion
1. Now, rename the first step of the training course to `Grab sphere`. Switch to the transitions tab, and add a single `Grab Object` condition to its transition to the second step. Drag the previously created `Sphere` into the object field named `Grabbable object`. Add a description:
```

    > This step will be completed when the trainee grabs the training scene object with the name `Sphere`, which has the `Grabbable Property` attached.

> 2. Open the second step in the [Step Inspector](../innoactive-creator/step-inspector.md). Rename it to `Bring to cube`. Add an `Object Nearby` condition to its transition to the third step. Set the `Range` to `1.5`, drag `Cube` and `Sphere` into the fields `First object` and `Second object`. Add a description:

Reviewer: ```suggestion
2. Open the second step in the [Step Inspector](../innoactive-creator/step-inspector.md). Rename it to `Bring to cube`. Add an `Object Nearby` condition to its transition to the third step. Set the `Range` to `1.5`, drag `Cube` and `Sphere` into the `First object` and `Second object` fields. Add a description:
```

Author: This was changed due to client fieedback that they found it confusing like it is right now. And this order is much more common.

#### Save and Load

The [Workflow window](../innoactive-creator/workflow-editor.md) and [Step Inspector](../innoactive-creator/step-inspector.md) automatically save all changes you make.

> You can open any selected training course by clicking the `Open in Workflow window` button in the `Inspector` tab of the `[TRAINING_CONFIGURATION]` game object. You can find this game object in your scene.

Reviewer: ```suggestion
You can open any selected training course by clicking the `Open in Workflow Window` button in the `Inspector` tab of the `[TRAINING_CONFIGURATION]` game object. You can find this game object in your scene.
```

Author: As this is a sentence, I would leave it minor.
